### PR TITLE
Make links relative in table of uploads

### PR DESCRIPTION
**Pull request by wd15**
Rather than always linking to pages.nist.gov, the links in the tables are now relative.

### CODE REVIEW

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1491.surge.sh

**Comment by wd15** [comment]
@tkphd: please review. Thanks.

The table now uses relative links, see

https://random-cat-1491.surge.sh/results/benchmark1a.1.ipynb/#Table-of-Results

For example, moose_1a_mpf links to

https://random-cat-1491.surge.sh/simulations/display/?sim=moose_1a_mpf

Note that some javascript is blocked on surge so that doesn't really load, but works on the [main page](https://pages.nist.gov/pfhub/simulations/display/?sim=moose_1a_mpf).

File: .github/workflows/surge.yml [hunk 1]
```diff
@@ -38,11 +38,11 @@ jobs:
         with:
           nix_path: nixpkgs=channel:nixos-unstable
 
-      - uses: cachix/cachix-action@v12
-        with:
-          name: pfhub-surge
-          extraPullNames: nix-community
-          skipPush: true
+      # - uses: cachix/cachix-action@v12
```
Comment (tkphd) [inline .github/workflows/surge.yml:41]: Temporarily stopping cachix, or a permanent removal?

**Review by tkphd — APPROVED** [review]
Can confirm: the table links now work, though blocked javascript on Surge means the summary pages there don't load anything. LGTM!

File: .github/workflows/surge.yml (hunk 1 above)
Comment (wd15) [reply .github/workflows/surge.yml]: Removed it now

**Comment by pfhub** [comment]
@wd15, the new version of the PFHub website is available at https://random-cat-1491.surge.sh

**Pull request merged**